app/views/configuration/show.html.haml

```diff
@@ -1 +1,2 @@
-#main_div= render :partial => @tabform
+- if %w(ui_1 ui_2 ui_3 ui_4 ui_5).include?(@tabform)
+  #main_div= render :partial => @tabform
```

Comment:
Another suggestion is to move this partial and associated Brakeman warning to a separate PR, thus allowing the other warnings that this PR addresses to be merged.